crux-mir/README.md

```diff
@@ -102,13 +137,62 @@ To compile and test a single Rust program:
 
 (Should print 2.)
 
+### Code coverage
+
+`crux-mir` has a code coverage tool that reports branches where only one side or the other was explored during symbolic execution. Note that due to how `mir-json` works, this will only report issues in functions that are called at least once from a `#[crux::test]` entrypoint. If you have dead code (i.e., functions that are never called by any entrypoint), then `mir-json` will discard the dead functions, which means that they won't be mentioned at all in the coverage output. Code coverage currently does not report the percentage of the coverage, only lists the paths that are not covered.
+
+* Clone this directory with `git clone https://github.com/GaloisInc/crucible.git` as we will need the `report-coverage` utility tool
+* In your crate run:
+  ```
+  $ crux-mir --lib --  --branch-coverage --path-sat --output-directory test-coverage
```

Comment:
Should this `--lib` flag be here? I think `--lib` is a flag meant to `cargo crux-test`, not `crux-mir` itself.

Alternatively, perhaps you meant to use `cargo crux-test --lib` instead of `crux-mir --lib`.